### PR TITLE
Update OR manual with info about manually re-routing AI trains

### PR DESCRIPTION
**Pull request by Csantucci**
Info refers to feature introduced with x1.3.1-58

### CODE REVIEW

File: Source/Documentation/Manual/driving.rst [hunk 1]
```diff
@@ -763,6 +763,45 @@ if the trains are far away, can lead to memory overflows.
...
+and then to get it back on the original route. The feature anyhow also allows to 
+re-route it without getting it back on the original route.
+
+It is suggested to look at this video which explains some practical case 
+https://youtu.be/-f0XVg7bSgU before continuing reading.
+
+To perform this correctly and in a way more similar to reality, some rules have to 
+be followed.
+The concept is that switches must be manually thrown only if they aren't reserved by a train. 
+To be sure of this it is necessary to force to stop the las signal(s) between train(s)
```
Comment (perpetualKid) [inline Source/Documentation/Manual/driving.rst]:
```suggestion
To be sure of this it is necessary to force to stop the **las** signal(s) between train(s) 
```
is this a typo?

Comment (Csantucci) [reply Source/Documentation/Manual/driving.rst]: Yes, it is. It must be corrected to "last" .

Comment (Csantucci) [reply Source/Documentation/Manual/driving.rst]: I'll prepare a further commit where I also add something else

**Comment by Csantucci** [comment]
Correction commit available.

**Pull request merged**